src/hooks/useProducts.js

```diff
@@ -38,5 +38,8 @@ export const useProducts = () => {
     };
   }, [storedValue]);
 
-  return { products, loading, error };
+  const updateProducts = (productsWithTimeFrameLabel) =>
```

Comment:
Where is this function being used? I didn't find it in other parts of the code.

Reply:
No 🤣 it's not being used, I already delete, thanks!